src/ServicePulse.Host/ServicePulse.Host.csproj

```diff
@@ -111,6 +111,9 @@
     <Content Include="app\js\services\factory.listener.js" />
     <Content Include="app\js\services\factory.notifier.js" />
     <Content Include="app\js\services\service.toast.js" />
+    <Content Include="app\js\views\message\controller.js" />
+    <Content Include="app\js\views\message\route.js" />
+    <Content Include="app\js\views\message\view.html" />
```

Comment:
Can we rename this to `messages-view.html` while we're at it and still thinking about https://github.com/Particular/ServicePulse/issues/400